app/templates/start-launch-eq.html

```diff
@@ -20,14 +20,9 @@ <h1>{{ _('Launch EQ') }}</h1>
         {{
             onsInput({
                 "type": "hidden",
-                "classes": "u-d-no",
                 "id": "token",
                 "name": "token",
-                "value": token,
-                "label": {
-                    "text": "Enter some text",
-                    "classes": "u-d-no"
-                }
+                "value": token
```

Comment:
When entering a UAC normally a respondent is redirected from the RH UI to EQ using the JWE in the URL protected by HTPS. Above is providing a page where the JWT token is stored in a hidden field. Should we ever store the JWT on the client side, is it at risk of being stolen, open other security problems? Is this page used, should we just remove it?